src/desktop/components/search_bar/view.coffee

```diff
@@ -85,6 +88,35 @@ module.exports = class SearchBarView extends Backbone.View
     @renderFeedback()
     @$el.addClass 'is-loading'
 
+  shouldMeasurePerformance: ->
+    performance? and sd.VOLLEY_ENDPOINT
+
+  recordPerformanceMeasurementStart: ->
+    return unless @shouldMeasurePerformance()
+
+    @performanceStart = performance.now()
+
+  reportPerformanceMeasurement: ->
+    return unless @shouldMeasurePerformance()
+
+    duration = performance.now() - @performanceStart
+    deviceType = if sd.IS_MOBILE then "mobile" else "desktop"
```

Comment:
i _think_ you can just hardcode this to 'desktop' since this component is only used on desktop. This is fine too tho!

Reply:
Good point! I'll switch this to be hard-coded.